data/tasks.txt

```diff
@@ -1,2 +1,2 @@
-[T] | [X] | seggs
-[D] | [X] | seggs | by: | today
+[T] | [ ] | seggs
+[T] | [ ] | seggs
```

Comment:
Good job on completing your tasks, but you might want to keep your tasks private 🤣 

Try adding `data/tasks.txt` to your `.gitignore` file!